crates/rpc/src/handlers/debug.rs

```diff
@@ -0,0 +1,120 @@
+use std::{collections::HashSet, sync::Arc};
+
+use actix_web::{
+    HttpResponse, Responder, get,
+    web::{Data, Path},
+};
+use hashbrown::HashMap;
+use ream_beacon_api_types::{
+    error::ApiError,
+    id::ID,
+    responses::{
+        BeaconHeadResponse, BeaconResponse, DataResponse, ForkChoiceNode, ForkChoiceResponse,
+        ForkChoiceValidity,
+    },
+};
+use ream_fork_choice::store::Store;
+use ream_operation_pool::OperationPool;
+use ream_storage::{db::ReamDB, tables::Field};
+
+use crate::handlers::state::get_state_from_id;
+
+#[get("/debug/beacon/states/{state_id}")]
+pub async fn get_debug_beacon_state(
+    db: Data<ReamDB>,
+    state_id: Path<ID>,
+) -> Result<impl Responder, ApiError> {
+    let state = get_state_from_id(state_id.into_inner(), &db).await?;
+
+    Ok(HttpResponse::Ok().json(BeaconResponse::new(state)))
```

Comment:
```suggestion
#[get("/debug/beacon/states/{state_id}")]
pub async fn get_debug_beacon_state(
    db: Data<ReamDB>,
    state_id: Path<ID>,
) -> Result<impl Responder, ApiError> {
    Ok(HttpResponse::Ok().json(BeaconResponse::new(get_state_from_id(state_id.into_inner(), &db).await?)))
```

we might as well inline this